scripts/superdesk-search/services/TagService.js

```diff
@@ -136,6 +137,23 @@ export function TagService($location, desks, userList, metadata, search, ingestS
         });
     }
 
+    function initRemovedFacets(params) {
```

Comment:
Can you please document this function? What does `initRemovedFacets` do exactly? For me, _"it initialises removed facets"_ doesn't quite describe the logic in there. I think it will help on the long run.